pkg/features/features_test.go

```diff
@@ -17,24 +17,26 @@
 package features_test
 
 import (
+	"testing"
+
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
 	"github.com/rancher/elemental-toolkit/pkg/features"
 )
 
+func TestTypes(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "config initializer test suite")
+}
+
 var _ = Describe("Init Action", func() {
 	Describe("Features", Label("features"), func() {
 		It("Returns an empty list of features", func() {
 			feats, err := features.Get([]string{})
 			Expect(err).ToNot(HaveOccurred())
 			Expect(feats).To(BeEmpty())
 		})
-		It("Returns all features", func() {
-			feats, err := features.Get([]string{})
-			Expect(err).ToNot(HaveOccurred())
-			Expect(len(feats)).To(Equal(len(features.All)))
-		})
```

Comment:
This test was essentially wrong, does the same exact thing as the previous one but expects a different output 😓 I guess this is an unnoticed leftover as the feature tests were not properly executed by ginkgo as they were not included in any test suite.